Makefile

```diff
@@ -23,6 +23,7 @@ BASE_IMAGE_SSH_PORT?=2222
 IMG_BUILDER=docker
 PLATFORMS ?= linux/amd64
 INTEL_PLATFORMS ?= linux/amd64
+MPICH_PLATFORMS ?= linux/amd64
```

Comment:
Can we build MPICH container images for linux/arm64 and linux/ppc64le?

Reply:
I have no experience with that, or a way of testing it. Other MPI implementations don't use it. So I would feel uneasy adding it without knowing it actually works.

Reply:
In OpenMPI, we are providing container images for linux/arm64 and linux/ppele64. here: https://hub.docker.com/layers/mpioperator/openmpi-builder/v0.4.0/images/sha256-e193e5365a2777d33d964f5563f4e5bdb7c7d1e5351c46dfbf7a436ea9338633?context=explore

So, supporting their platforms would be good.

Reply:
If you don't have environments for testing them, creating an issue to keep tracking them would be well.

Reply:
Sure. I won;t be adding support for those architectures, I I've created the issue, as suggested: https://github.com/kubeflow/mpi-operator/issues/565